lib/server.js

```diff
@@ -253,8 +253,9 @@ function Server(options) {
     this.router = options.router;
     this.routes = {};
     this.secure = false;
-    this.versions = options.versions || options.version || [];
     this.socketio = options.socketio || false;
+    this._once = (options.strictNext === false) ? once : once.strict;
```

Comment:
at this point since you're in here anyway, might want to drop the `once` dependency and just use the function provided by `lodash`